lib/re.js

```diff
@@ -78,6 +78,10 @@ module.exports = function (opts) {
 
     '[\\-;:&=\\+\\$,\\"\\.a-zA-Z0-9_]+';
 
+  re.src_get_params =
+
+    '((?:(?:\\?|\&)(?:[^=]+)\=(?:[^&]+))[^\\s\\n\\r\\)]+|)';
+
```

Comment:
That's very suspicious.

- I would restrict param names with predefined values (subject|body|...). We should be sure that that false positive is not possible.
- It's good to break such equations to multiple strings with comments (if needed). Right now - very difficult to read.
- Not sure about condition when tail should be terminated.
- `[^\\s\\n\\r\\)]` hand-made !src_ZPCc ?